File: Invite/UIView+Invite.m
//
//  UIView+Invite.m
//  Invite
//
//  Created by Ryan Jennings on 4/6/15.
//  Copyright (c) 2015 Appuous. All rights reserved.
//

#import "UIView+Invite.h"

@implementation UIView (Invite)

+ (void)appearanceObjC
{
    [[UIButton appearanceWhenContainedIn:[UINavigationBar class], nil] setBackgroundColor:[UIColor clearColor]];
    [[UIButton appearanceWhenContainedIn:[UISearchBar class], nil] setBackgroundColor:[UIColor clearColor]];
    [[UIButton appearanceWhenContainedIn:[UIToolbar class], nil] setBackgroundColor:[UIColor clearColor]];
    [[UIButton appearanceWhenContainedIn:[UITableViewCell class], nil] setBackgroundColor:[UIColor clearColor]];
}

@end

File: Invite/UIView+Invite.swift
//
//  UIView+Invite.swift
//  Invite
//
//  Created by Ryan Jennings on 10/10/15.
//  Copyright © 2015 Appuous. All rights reserved.
//

import UIKit

extension UIView
{
    func roundCorners(corners:UIRectCorner, radius: CGFloat)
    {
        let path = UIBezierPath(roundedRect: self.bounds, byRoundingCorners: corners, cornerRadii: CGSize(width: radius, height: radius))
        let mask = CAShapeLayer()
        mask.path = path.CGPath
        self.layer.mask = mask
    }
}
